### PR TITLE
Create initial principles from Presto GB feedback

### PR DESCRIPTION
**Pull request by caniszczyk**
Signed-off-by: Chris Aniszczyk <caniszczyk@linuxfoundation.org>

### CODE REVIEW

File: PRINCIPLES.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## 2) Open, transparent technical leadership and direction
+
+At the end of the day, technology and technical direction is what keeps a community interested and the innovation going. We want the project to be inclusive and the technical roadmap to be fairly representative of the community’s view. In fact, the TSC has its own open charter and the meetings are open for everyone to attend. We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges. 
+
+## 3) No one individual or company is greater than the team and project
```
Comment (bhenhsi) [inline PRINCIPLES.md]: team and project -> the project and its community

File: PRINCIPLES.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Presto Foundation Principles
+
+## 1) One open, neutral and united Presto community 
+
+One of the main reasons the founding members came together and created the Presto Foundation was to unite the Presto community under the original project. The Presto community and it’s contributors have been working on many great capabilities and connectors, and many of these haven’t made their way back into the repo, and we truly believe that these can benefit many other users. We invite all Presto community members to embrace PrestoDB and contribute back to move the project faster, reduce duplication of efforts and create wider impact for the entire community.
```
Comment (bhenhsi) [inline PRINCIPLES.md]: I'd suggest removing "under the original project".

File: PRINCIPLES.md (hunk 1 above)
Comment (briankhsieh) [reply PRINCIPLES.md]: (To me it genal if we are using this articulate he principle. )
We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.  
-> 
We invite you to join the TSC meetings. We believe under the open and transparent governance of TSC, the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.

File: PRINCIPLES.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## 2) Open, transparent technical leadership and direction
+
+At the end of the day, technology and technical direction is what keeps a community interested and the innovation going. We want the project to be inclusive and the technical roadmap to be fairly representative of the community’s view. In fact, the TSC has its own open charter and the meetings are open for everyone to attend. We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.
```
Comment (bhenhsi) [inline PRINCIPLES.md]: (To make it genal if we are using this articulate he principle. )
We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.
->
We invite you to join the TSC meetings. We believe under the open and transparent governance of TSC, the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.

File: PRINCIPLES.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## 3) No one individual or company is greater than the team and project 
+
+The Linux Foundation is dedicated to building sustainable ecosystems around open source projects to accelerate technology development and commercial adoption. It is the home of Linux creator Linus Torvalds, and most importantly provides a neutral home to Linux and other projects where their development can be protected and accelerated for years to come. This important principle of neutrality has been the core of Linux Foundation as well as Apache Software Foundation so that no one individual or company agenda becomes that of the projects. This core neutrality principle is rarely something embraced by foundations created by other companies or individuals. Neutrality will drive the approach to governance that we have adopted so that no single company or individual can take undue influence.
```
Comment (bhenhsi) [inline PRINCIPLES.md]: I'd suggest removing "This core neutrality principle is rarely something embraced by foundations created by other companies or individuals".

File: PRINCIPLES.md (hunk 3 above)
Comment (nezihyigitbasi) [reply PRINCIPLES.md]: Typo in my last name `Yigitbai` -> `Yigitbasi`

Comment (nezihyigitbasi) [reply PRINCIPLES.md]: I would say "solve some hard technical challenges" -> "continue solving hard technical problems"

Comment (nezihyigitbasi) [reply PRINCIPLES.md]: It would be good to link to tsc repo in this 2. item.

Comment (caniszczyk) [reply PRINCIPLES.md]: fixed based on Brian's suggestion so we avoided the last name spelling issue

**Comment by caniszczyk** [comment]
updated changes RFC

**Comment by caniszczyk** [comment]
Approved by the Presto Foundation GB on 12/16

**Pull request merged**